### PR TITLE
replace wordings of WIP by draft PR

### PR DESCRIPTION
**Pull request by ickc**
Improve this pull request template by mentioning draft PR.

### CODE REVIEW

**Comment by ickc** [comment]
Based on the automatic message from @esc in another PR it seems it is parsing the [WIP] label so may be this PR itself is not self-contained.

**Comment by esc** [comment]
> Based on the automatic message from @esc in another PR it seems it is parsing the [WIP] label so may be this PR itself is not self-contained.

<joke> I am (still) human, but I do want to become an AI at some point by transferring my consciousness into a computer -- so thank you for assuming I am a bot, it confirms that I may be part of the way there already!!!</joke>


Jokes aside, I'll mark this as a feature request also. Do you want it to be ready for review also?

**Comment by esc** [comment]
> Jokes aside, I'll mark this as a feature request also. Do you want it to be ready for review also?

Sorry, I meant "review effort -short".

**Comment by ickc** [comment]
Ha! I thought you wrote a bot using your API keys showing you as the author...

It's a very minor wording change so it's ready to review. Feel free to reject it though if you think the current one is fine.

**Comment by esc** [comment]
> Ha! I thought you wrote a bot using your API keys showing you as the author...

I should totally do that and call it "Augmented Intelligence" I'll get it learn from all my comments on github and it will generate responses in my communication style! 

> It's a very minor wording change so it's ready to review. Feel free to reject it though if you think the current one is fine.

OK.

**Comment by esc** [comment]
> It's a very minor wording change so it's ready to review. Feel free to reject it though if you think the current one is fine.

I think it is valid, we are adopting the builtin WIP feature more and more, we should encourage folks to use it! Thank you for the patch!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -18,8 +18,9 @@ Here's some guidelines to help the review process go smoothly.
    noted here: https://help.github.com/articles/closing-issues-using-keywords/
 
 3. If your pull request is not ready for review but you want to make use of the
-   continuous integration testing facilities please label it with `[WIP]` and
-   then remove the label when you'd like it to be reviewed.
+   continuous integration testing facilities please click the arrow besides
```
Comment (esc) [inline .github/PULL_REQUEST_TEMPLATE.md:21]: I am sure there must be a comma here somewhere?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -18,8 +18,9 @@ Here's some guidelines to help the review process go smoothly.
...
+   "Create Pull Request" and choose "Create Draft Pull Request". And
```
Comment (esc) [inline .github/PULL_REQUEST_TEMPLATE.md:22]: I am not sure it is good style to begin a sentence with `And`, you can probably drop that?

**Review by esc — CHANGES_REQUESTED** [review]
@stuartarchibald there are some finer grammatical points in this PR that require an english native speaker, can you take a look please? Thank you!

**Comment by esc** [comment]
@ickc I have called on @stuartarchibald to help with the grammatical nuances, let's await their response.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ickc) [reply .github/PULL_REQUEST_TEMPLATE.md]: resolved in commit 0288022

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (ickc) [reply .github/PULL_REQUEST_TEMPLATE.md]: resolved in commit 0288022

**Review by esc — APPROVED** [review]

**Comment by esc** [comment]
@ickc thank you!

**Pull request merged**